src/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.main-container {
+  width: 50%;
+  margin: 20px auto;
+  box-shadow: 1px 2px 5px 5px rgb(185, 183, 183);
+  border-radius: 10px;
+}
+
+.title {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: 1px solid rgb(214, 212, 212);
+}
+
+#rotate-arrow {
+  width: 20px;
+  height: 20px;
+  padding-right: 10px;
+}
+
+h3 {
+  padding: 10px;
+}
+
+input[type="text"] {
+  display: flex;
+  width: 100%;
+  font-style: italic;
+  font-size: 15px;
+  border: none;
+  outline: none;
+}
+
+#form {
+  display: flex;
+  padding: 15px;
+  border-bottom: 1px solid rgb(214, 212, 212);
+}
+
+ul {
+  list-style-type: none;
+}
+
+ul li {
+  display: flex;
+  width: 100%;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  border-bottom: 1px solid rgb(214, 212, 212);
+
+  /* font-size: 15px; */
```

Comment:
* Kindly remove all commented code from your codebase as it makes your code look untidy and unprofessional 🙏